_includes/contributor-carousel-selection.html

```diff
@@ -87,7 +87,7 @@
                     <div class="card-body text-center py-0">
                         <p class="card-title">{{ contributor }}</p>
                         {%- if contributors[contributor].affiliation %}
-                        <p class="card-affiliation">{{ contributors[contributor].affiliation }}</p>
+                        <p class="card-affiliation">{{ contributors[contributor].affiliation | join: ", " }}</p>
```

Comment:
I personally like more the backslashes since they are visually more separating (see my example and other instances like the RDMkit). Do you have a strong preference for the comma?

Reply:
No preference @bedroesb - happy for you to make the call and also with "/" it would be backwards compatible with that you have now.

Reply:
Feel free to change that line and commit if you like.